redis/reply_test.go

```diff
@@ -225,6 +225,130 @@ func TestSlowLog(t *testing.T) {
 	}
 }
 
+func TestLatency(t *testing.T) {
+	c, err := dial()
+	if err != nil {
+		t.Errorf("TestLatency failed during dial with error " + err.Error())
+		return
+	}
+	defer c.Close()
+
+	resultStr, err := redis.Strings(c.Do("CONFIG", "GET", "latency-monitor-threshold"))
+	if err != nil {
+		t.Errorf("TestLatency failed during CONFIG GET latency-monitor-threshold with error " + err.Error())
+		return
+	}
```

Comment:
As above, more below.
```suggestion
	require.NoError(t, err)
```
alternative:
```suggestion
	if err != nil {
		t.Fatalf("CONFIG GET latency-monitor-threshold: %s", err)
	}
```